src/app/components/Metadata/index.jsx

```diff
@@ -38,6 +38,7 @@ const Metadata = ({
       <meta name="article:author" content={articleAuthor} />
       <meta name="article:modified_time" content={timeLastUpdated} />
       <meta name="article:published_time" content={timeFirstPublished} />
+      <meta name="Description" content={description} />
```

Comment:
The value for the name attribute should be lowercase: `description`. 
Also, it would be good to keep the meta tags in alphabetic order.